layouts/BlogLayout.js

```diff
@@ -0,0 +1,96 @@
+import Link from '@/components/Link'
+import Tag from '@/components/Tag'
+import siteMetadata from '@/data/siteMetadata'
+import { useState } from 'react'
+import Pagination from '@/components/Pagination'
+
+const postDateTemplate = { year: 'numeric', month: 'long', day: 'numeric' }
+
+export default function BlogLayout({ posts, title, pagination, prevDisable, nextDisable }) {
```

Comment:
I actually think we do not a new layout and can use the prev one. Just let `pagination` be an optional variable and conditionally render the `Pagination` component. For the tag pages with no pagination passed it, it will display the full list with no pagination (i.e. same behaviour as previously).

Reply:
I conditionally rendered the `Pagination` component this using `next/router` and `router.pathname.indexOf('/blog') > -1`